src/aura/HH_AddressMgr/HH_AddressMgr.css

```diff
@@ -0,0 +1,11 @@
+.THIS li.selected {
+    background-color: token(colorBackgroundRowSelected);
```

Comment:
I'm in a dev org I've had for years.  not GS0.  can't tell what server because I've got custom domains enabled!  I needed a way to get the default selected item color, and tokens is where I found it.  since we are using our own static resource for SLDS, I think we are safe from changes, until we upgrade of course.